.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,37 @@
+name: Check Build
+
+on:
+  pull_request:
+  push:
+    branches:
+      - develop
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout 🛎
+        uses: actions/checkout@v3
+
+      - name: Setup node env 🏗
+        uses: actions/setup-node@v3
+        with:
+          node-version: 10.7
+            
+      - name: Cache node_modules 📦
```

Comment:
We can't cache on setup-node step ??
https://github.com/actions/setup-node#caching-global-packages-data